lib/Sema/DerivedConformances.cpp

```diff
@@ -330,6 +329,13 @@ ValueDecl *DerivedConformance::getDerivableRequirement(TypeChecker &tc,
     if (name.isSimpleName(ctx.Id_Parameters))
       return getRequirement(KnownProtocolKind::Parameterized);
 
+    // SWIFT_ENABLE_TENSORFLOW
+    // Parameterized.TangentVector and Parameterized.CotangentVector
+    if (name.isSimpleName(ctx.Id_TangentVector) ||
+        name.isSimpleName(ctx.Id_CotangentVector))
+      // TOOD: Also `getRequirement(KnownProtocolKind::Differentiable)`?
+      return getRequirement(KnownProtocolKind::Parameterized);
```

Comment:
Similarly, you can remove this code. This will fall out of `Differentiable` derived conformances.